api/HTMLElement.json

```diff
@@ -841,10 +841,12 @@
               "version_added": false
             },
             "opera": {
-              "version_added": null
+              "version_added": "≤12.1",
+              "version_removed": "15"
```

Comment:
This seems inconsistent with the data for Chrome. If `dropzone` was removed in Chrome 59, would it not have been in Opera 15?

Reply:
Surprisingly enough, it was not introduced to Opera Blink.

Reply:
Huh. I guess it might have required some platform integration work which was never done and the feature was disabled. Thanks for confirming!